sctipt.py

```diff
@@ -0,0 +1,206 @@
+#! /usr/bin/env python
+
+# !git clone https://github.com/ToluClassics/LowResourceOCR.git 
+
+
+# !pip install -q transformers
+# !pip install -q datasets jiwer
+import os, argparse
+import sys
+import pandas as pd
+from sklearn.model_selection import train_test_split
+
+import torch
+from torch.utils.data import Dataset, DataLoader
+
+from PIL import Image
+from tqdm.notebook import tqdm
+
+from transformers import TrOCRProcessor, VisionEncoderDecoderModel, AdamW
+
+from datasets import load_metric
+
+"""
+features we may want to pass as arguments include:
+text_path, 
+ImageFolder_Path, 
+split_size, 
+max_target_length, 
+no_epochs,
+train_batch_size
+"""
+
+device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+
+# Create the parser
+the_parser = argparse.ArgumentParser(description='List the content of a folder')
+
+# Add the arguments
+the_parser.add_argument('--text_path',
+                       metavar='path',
+                       type=str,
+                       help='the path to dataframe .txt file')
+the_parser.add_argument('--dir_path',
+                       metavar='dpath',
+                       type=str,
+                       help='the path to dataframe .txt file')
+the_parser.add_argument('--split_size',
+                       type=float,
+                       help='train_test split size for validation',
+                       default=0.2)
+the_parser.add_argument('--max_target_length',
+                       type=int,
+                       help='train_test split size for validation',
+                       default=256)
+the_parser.add_argument('--no_epochs',
+                       type=int,
+                       help='no of epochs in training',
+                       default=10)
+the_parser.add_argument('--training_batch_size',
+                       type=int,
+                       help='training batch size',
+                       default=4)
```

Comment:
then we can put these arguments in a function just like here

https://github.com/castorini/afriteva/blob/main/scripts/sample_tokenizer_sentences.py